test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-link_directories(/src/LPM/src /src/LPM/src/libfuzzer /src/LPM/external.protobuf/lib)
+if (OSSFUZZ)
+    link_directories(/src/LPM/src /src/LPM/src/libfuzzer /src/LPM/external.protobuf/lib)
+endif()
```

Comment:
Earlier we neither compiled nor linked ossfuzz harnesses. Now we compile them but don't link.